src/mapping/nav2_map_server/CMakeLists.txt

```diff
@@ -0,0 +1,88 @@
+cmake_minimum_required(VERSION 3.5)
+project(nav2_map_server)
+
+set(YAML_CPP_ROS2_LIBRARIES $ENV{HOME}/ros2_ws/ros2_ws/install/rviz_yaml_cpp_vendor/opt/rviz_yaml_cpp_vendor/lib/libyaml-cpp.so)
+set(YAML_CPP_ROS2_INCLUDE_DIRS $ENV{HOME}/ros2_ws/ros2_ws/install/rviz_yaml_cpp_vendor/opt/rviz_yaml_cpp_vendor/include)
+
+# Default to C++14
+if(NOT CMAKE_CXX_STANDARD)
+  set(CMAKE_CXX_STANDARD 14)
+endif()
+
+if(CMAKE_COMPILER_IS_GNUCXX OR CMAKE_CXX_COMPILER_ID MATCHES "Clang")
+  add_compile_options(-Wall -Wextra -Wpedantic)
+endif()
+
+find_package(ament_cmake REQUIRED)
+find_package(rclcpp REQUIRED)
+find_package(nav_msgs REQUIRED)
+find_package(std_msgs REQUIRED)
+find_package(rmw REQUIRED)
+find_package(Bullet REQUIRED)
+find_package(SDL REQUIRED)
+find_package(SDL_image REQUIRED)
+#find_package(yaml-cpp REQUIRED)
+find_package(example_interfaces REQUIRED)
+
+find_package(PkgConfig REQUIRED)
+pkg_check_modules(YAMLCPP yaml-cpp REQUIRED)
+
+include_directories(
+  include
+  ${BULLET_INCLUDE_DIRS}
+  ${ament_INCLUDE_DIRS}
```

Comment:
This shouldn't be necessary. It should be covered by the `ament_target_dependencies` function